robottelo/cli/factory.py

```diff
@@ -2468,6 +2471,44 @@ def setup_org_for_a_rh_repo(options=None):
     }
 
 
+def setup_org_for_a_rh_repo(options=None, force_manifest_upload=False):
```

Comment:
we can add a kwarg force_use_cdn=False for the tests that need only cdn like "test_positive_add_redhat_product"
that will call this function with force_use_cdn=True instead of using private function